src/test/java/com/groupseven/pdfproject/PageModelTest.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package com.groupseven.pdfproject;
+
+import javafx.scene.layout.VBox;
+import org.junit.After;
+import org.junit.AfterClass;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import static org.junit.Assert.*;
+
+/**
+ *
+ * @author dwkal
+ */
+public class PageModelTest {
+
+    public PageModelTest() {
+    }
+
+    @BeforeClass
+    public static void setUpClass() {
+    }
+
+    @AfterClass
+    public static void tearDownClass() {
+    }
+
+    @Before
+    public void setUp() {
+    }
+
+    @After
+    public void tearDown() {
+    }
+
+    /**
+     * Test of getNode method, of class PageModel.
+     */
+    @Test
+    public void testGetNode() {
+        System.out.println("getNode");
+        PageModel instance = null;
+        VBox expResult = null;
+        VBox result = instance.getNode();
```

Comment:
as instance is declared as null, this is causing a null pointer exception that also causes your tests to fail.